docs/central.md

```diff
@@ -321,11 +327,15 @@ by adding an extra parameter in the DSL or setting a Gradle property
     SONATYPE_AUTOMATIC_RELEASE=true
     ```
 
-Regardless of whether it is done automatically or manually after the staging repository is released the artifacts
-will be synced to Maven Central. This process takes 10-30 minutes and when it is completed
-the artifacts are available for download.
+### Manual release
+
+The release (step 4) can be done manually by running the following command, so that the plugin will
+only do step 1 to 3:
+```
+./gradlew publishAllPublicationsToMavenCentralRepository --no-configuration-cache
+```
 
-## Timeouts
+### Timeouts
```

Comment:
Intended change?

Reply:
Yes, these timeouts only affect publishing releases so it makes sense to move it under that.